db/migrate/20150129082106_add_from_quote_to_topic_link.rb

```diff
@@ -0,0 +1,5 @@
+class AddFromQuoteToTopicLink < ActiveRecord::Migration
+  def change
+    add_column :topic_links, :from_quote, :boolean, default: false, null: false
```

Comment:
I dunno, this is adding a feature we don't really need to add just to fix a bug, can we just fix bug instead?